packages/winpanda/extra/src/winpanda/core/cmdconf.py

```diff
@@ -354,30 +319,120 @@ def get_dcos_conf(self):
+        :param dstor_lpi_path:         str, URL path to the DC/OS Linux package
+                                       index object at the DC/OS distribution
+                                       storage
+        :return dstor_dcoscfgpkg_path: str, URL path to the DC/OS aggregated
+                                       config package at the DC/OS distribution
+                                       storage
+        """
+        dcos_conf_pkg_name = 'dcos-config-win'
+
+        # Linux package index direct URL
+        lpi_url = posixpath.join(dstor_root_url, dstor_lpi_path)
+        lpi_fname = Path(dstor_lpi_path).name
+
+        try:
+            cm_utl.download(lpi_url, str(self.inst_storage.tmp_dpath))
+            LOG.debug(f'{self.msg_src}: DC/OS Linux package index: Download:'
+                      f' {lpi_fname}: {lpi_url}')
+        except Exception as e:
+            raise cr_exc.RCDownloadError(
+                f'DC/OS Linux package index: {lpi_fname}:'
+                f' {lpi_url}: {type(e).__name__}: {e}'
+            ) from e
+
+        lpi_fpath = self.inst_storage.tmp_dpath.joinpath(lpi_fname)
+
+        try:
+            lpi = cr_utl.rc_load_json(lpi_fpath,
+                                      emheading='DC/OS Linux package index')
+
+            if not isinstance(lpi, dict):
+                raise cr_exc.RCInvalidError(
+                    f'DC/OS Linux package index: {lpi_fpath}: Invalid'
+                    f' structure: {lpi}'
+                )
+
+            dcos_conf_pkg_desc = lpi.get(dcos_conf_pkg_name)
+
+            if dcos_conf_pkg_desc is None:
+                raise cr_exc.RCElementError(
+                    f'DC/OS Linux package index: {lpi_fpath}: DC/OS aggregated'
+                    f' config package descriptor is missed:'
+                    f' {dcos_conf_pkg_name}: {lpi}'
+                )
+
+            if not isinstance(dcos_conf_pkg_desc, dict):
+                raise cr_exc.RCElementError(
+                    f'DC/OS Linux package index: {lpi_fpath}: Invalid DC/OS'
+                    f' aggregated config package descriptor:'
+                    f' {dcos_conf_pkg_desc}'
+                )
+
+            dstor_dcoscfgpkg_path = dcos_conf_pkg_desc.get('filename')
+            if dstor_dcoscfgpkg_path is None:
+                raise cr_exc.RCElementError(
+                    f'DC/OS Linux package index: {lpi_fpath}: DC/OS aggregated'
+                    f' config package descriptor: Distribution storage path is'
+                    f' missed: {dcos_conf_pkg_desc}'
+                )
+            if not isinstance(dstor_dcoscfgpkg_path, str):
+                raise cr_exc.RCElementError(
+                    f'DC/OS Linux package index: {lpi_fpath}: DC/OS aggregated'
+                    f' config package descriptor: Distribution storage path:'
+                    f' Invalid type: {dstor_dcoscfgpkg_path}'
+                )
+        except Exception:
+            raise
```

Comment:
What are you trying to do here? These lines do nothing.

Reply:
Fixed in the last PR update